### DISCUSSION TITLE
Using <Del> as VM_leader, possible?

### DISCUSSION THREAD

**Discussion by IndianBoy42** [mg979/vim-visual-multi#237] [Q&A] [answered]
I want to use `<del>` as my leader key for visual-multi, I've got a weird ergo split keyboard with layers and suffice to say this is a pretty convenient key to use and I don't have any need for the builtin functionality. 

The problem is I don't know how to disable the builtin functionality. Outside of VM mode `<Del>` works as the leader but in VM mode when I hit `<Del>` it just deletes the characters at each cursor. Actually if I hit the full sequence quick enough I can access the VM functions, so I guess its a timeout problem. But as I said, I have no need for `<del>` as deleting characters, x exists

**Comment by IndianBoy42** [comment]
I found another similar problem, which is that I wanted to map Transpose to `(` but VM binds that to `VM-Motion-(` and seemingly does that later than Transpose is mapped, so I am not able to use Transpose. There doesn't seem to be a way for me to remove `Motion-(` from being mapped

**Comment by mg979** [comment, marked as answer]
The settings and mapping system are meant for general use, probably not for certain use cases. But there are autocommands and functions that are called when VM starts and exits, you can use them to override certain mappings if you know what you're doing, then you would need to unmap stuff on exit manually. Check the documentation.
